internal/services/storage/client/client.go

```diff
@@ -24,6 +25,7 @@ type Client struct {
 	StorageDomainSuffix string
 
 	ResourceManager *storage_v2023_01_01.Client
+	StorageClient   *storage_v2023_05_01.Client
```

Comment:
Instead of introducing a new field same as “ResourceManager” but with a different api version, could you please submit a new pr to upgrade the api version for the storage service to 2023-05-01 first? Otherwise, you can just use the 2023-01-01 one?

Reply:
Hi @magodo  thanks for the feedback.
I will use the version 2023-01-01 instead of the updated one.